crates/symbolicator/src/config.rs

```diff
@@ -243,9 +243,15 @@ pub struct Config {
     /// Number of subprocesses in the internal processing pool.
     pub processing_pool_size: usize,
 
-    /// The timeout for downloads.
+    /// The maximum timeout for downloads.
     #[serde(with = "humantime_serde")]
-    pub download_timeout: Duration,
+    pub max_download_timeout: Duration,
+
+    /// The timeout per GB for streaming downloads.
+    ///
+    /// We default this to half of the `max_download_timeout` duration, assuming a maximum download size of 2GB.
```

Comment:
```suggestion
    /// When the download size is known this is the duration per-gigabyte, thus for smaller files resulting in a faster timeout than the upper limit set by `max_download_duration`.
```

I was also going to write something about when the download size is not know to match your current sentence.  But does this really matter?  Isn't it "when the download size is not known this is ignored and only `max_download_timeout` is used"?  I mean, isn't that the same thing but more easily expressed?  Or am I getting this completely wrong?